### PR TITLE
Update/list post

### PR DESCRIPTION
**Pull request by yoslie**
### Overview
update revision testing list post

### Changes
- change popover on popup action list post due to bug clickaway
- handle numbering item list post on change page

### Related
- [T041 - [CMS - Post] Nomor urut paginasi list data](https://app.clickup.com/t/9003239225/CES-3749)
- [T042 - [CMS - Post] Cek hamburger menu list data](https://app.clickup.com/t/9003239225/CES-3750)

### Evidence
title: Update post list (handle numbering item on change page, change popover on popup action)
project: J-Site
participants: @yoslie @Ibwedagama @ayocodingit @doohanas @marsellavaleria19

### CODE REVIEW

File: components/Post/ItemAction.vue [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+  </Popover>
 </template>
 
 <script setup lang="ts">
-  const isOpenListAction = ref(false)
-
-  function toggleOpenModal() {
-    isOpenListAction.value = !isOpenListAction.value
-  }
+  import { Popover, PopoverButton, PopoverPanel } from '@headlessui/vue'
```
Comment (Ibwedagama) [inline components/Post/ItemAction.vue:31]: why are we still using headless UI @yoslie? did Nuxt UI doesn't provide the same component?

File: pages/posting/arsip.vue [hunk 2]
```diff
@@ -101,14 +101,32 @@
...
+      if (metaData?.page === 1) {
+        post.data = postData.map((item, index) => ({
+          ...item,
+          index: index + 1,
+        }))
+      } else {
+        post.data = postData.map((item, index) => ({
+          ...item,
+          index:
+            index + 1 + (Number(metaData?.page) - 1) * Number(metaData?.limit),
```
Comment (Ibwedagama) [inline pages/posting/arsip.vue:124]: hello @yoslie why do we have to manually assign the index like this? is there an issue with API response?

File: components/Post/ItemAction.vue (hunk 1 above)
Comment (Ibwedagama) [reply components/Post/ItemAction.vue:31]: oh, there is a bug with Nuxt UI clickaway?  🤔

Comment (yoslie) [reply components/Post/ItemAction.vue:31]: somehow the clickaway not worked and it caused bug like this
image

File: pages/posting/arsip.vue (hunk 2 above)
Comment (yoslie) [reply pages/posting/arsip.vue:124]: API response doesn't provide the data numbering 😭 
this index used for item numbering @Ibwedagama

File: components/Post/ItemAction.vue (hunk 1 above)
Comment (yoslie) [reply components/Post/ItemAction.vue:31]: and I cant find the documentation about the clickaway configuration in NuxtUI [UPopover](https://ui.nuxt.com/overlays/popover) component

File: pages/posting/arsip.vue (hunk 2 above)
Comment (Ibwedagama) [reply pages/posting/arsip.vue:124]: ah I see, so when we change the page the numbering will reset to 1, 2, 3, 4 ... so we have to do it manually @yoslie

**Review by Ibwedagama — APPROVED** [review]

**Review by marsellavaleria19 — APPROVED** [review]

File: pages/posting/arsip.vue (hunk 2 above)
Comment (ayocodingit) [reply pages/posting/arsip.vue:124]: For the numbering, You can use `from` on the meta response, yeah

**Review by doohanas — APPROVED** [review]

File: pages/posting/arsip.vue (hunk 2 above)
Comment (yoslie) [reply pages/posting/arsip.vue]: ah I see that's more efficient, thanks @ayocodingit .. updated [here](https://github.com/jabardigitalservice/j-site-builder/pull/87/commits/e7d89f9e9edcec62bf0fb741f9be107aaea5fd99)

**Review by ayocodingit — APPROVED** [review]

**Pull request merged**